package.json

```diff
@@ -351,6 +354,7 @@
     "react-native-fs": "^2.20.0",
     "react-native-gesture-handler": "^1.10.3",
     "react-native-get-random-values": "^1.8.0",
+    "react-native-google-acm": "git+https://github.com/Web3Auth/react-native-google-acm.git#3ad58f4c11273ba102ede93d2a3148e45c84d248",
```

Comment:
`Did we bypass the paid version by forking our own and adding our own implementation?`
Yes, we use react-native wrapper to call google account credentials manager for android seamless login